requirements-types.txt

```diff
@@ -7,6 +7,7 @@
 --requirement docs/sphinx_api_docs_source/requirements-dev-api-docs.txt
 # typing stubs
 pandas-stubs
+sqlalchemy<1.6.0
```

Comment:
Does this only fail on the marker-check step or also in our integration tests?

Reply:
integration test don't run when the lint step fails, so i'm not sure

Reply:
This is how I got past it, but also haven't seen the integration tests run to completion yet. I suspect it may still fail there.

https://github.com/great-expectations/great_expectations/pull/10095/files#r1671350671